arrow/util.py

```diff
@@ -46,6 +52,15 @@ def is_timestamp(value):
         return False
 
 
+def is_ordinal(value):
+    """Check if value is a valid ordinal."""
+    if isinstance(value, bool) or not isinstance(value, int):
+        raise TypeError(f"ordinal must be an intger (got type {type(value)})")
```

Comment:
Typo, but this won't be necessary if you implement the boolean change I recommended.

Reply:
@pgogri there is a typo here in "intger".

Also, let's keep error messages consistent: capital first letter and end them with a period :).

Reply:
That is sloppy at best! thanks for pointing that out.